### ISSUE TITLE
Erroneous CoD for hsp_hs_demo?

### ISSUE THREAD

**Issue by ajraymond** [bluekitchen/btstack#176]
I've noticed that the CoD used in hsp_hs_demo.c is 0x240404.

This CoD doesn't work with an AG I am using, while 0x200404 does (and my MPOW headset does advertise itself as 0x200404).

This extra bit corresponds to "Rendering" service class.

**Comment by mringwal** [comment]
Does your AG connect to hfp_hsp_demo with the CoD set to?

**Comment by ajraymond** [comment]
Yes, it connects to the hfp_hsp_demo when the CoD is set to 0x200404
